solr/solr-ref-guide/modules/deployment-guide/pages/backup-restore.adoc

```diff
@@ -452,7 +452,7 @@ An example configuration using these properties can be found below:
 [source,xml]
 ----
 <backup>
-  <repository name="hdfs" class="org.apache.solr.core.backup.repository.HdfsBackupRepository" default="false">
+  <repository name="hdfs" class="org.apache.solr.hdfs.backup.repository.HdfsBackupRepository" default="false">
```

Comment:
We need to add this in the `major-changes-in-solr-9.adoc`

Reply:
Addressed in [89993fd](https://github.com/apache/solr/pull/688/commits/89993fd056d8cdd8f752805b80968f371f5f2d2c)